tf-psa-crypto/tests/suites/test_suite_psa_crypto.function

```diff
@@ -10162,8 +10165,20 @@ void generate_key(int type_arg,
         goto exit;
     }
 
+    num_ops_prior = psa_generate_key_iop_get_num_ops(&operation);
+    TEST_EQUAL(num_ops_prior, 0);
+
     do {
         status = psa_generate_key_iop_complete(&operation, &iop_key);
+
+        if (status == PSA_SUCCESS || status == PSA_OPERATION_INCOMPLETE) {
+            num_ops = psa_generate_key_iop_get_num_ops(&operation);
```

Comment:
Preexisting: our current implementation of key generation doesn't do any ECC arithmetic, just generating one random integer and doing some minor work. This is less work than what we consider one op, which is a multiplication on P256. So we could reasonably say that key generation takes 0 ops, which would cause this test to fail. We [set the number of ops to 1](https://github.com/Mbed-TLS/mbedtls/blob/f7ebd7375558dde1d8d55fe56c7604760b3213bd/tf-psa-crypto/drivers/builtin/src/psa_crypto_ecp.c#L648), which is also a reasonable choice (rounding up instead of down). I would like an explanation of why we made that choice, and how it's relevant here.

Here, we should have a comment to explain how we assume that key generation takes at least 1 op and how it simplifies our testing, but it would be ok to allow generation to take 0 ops if it returns on the first call to complete(). This might be necessary if we ever add driver support.

In `mbedtls_psa_ecp_generate_key_iop_complete`, it would be good to explain why we chose 1. The fact that 1 makes testing simpler than 0 is a good reason, we should just write it down.